typescript/test/test_application/showcase-setup.sh

```diff
@@ -0,0 +1,22 @@
+mkdir showcase-server 
+cd showcase-server
+export GAPIC_SHOWCASE_VERSION=0.5.0
+export OS=`uname`
```

Comment:
Let's do `uname | tr A-Z a-z` to make sure it's lowercase. On my Mac, `uname` gives `Darwin` with the capital D (but if you verified it works, we can leave it as is)

Reply:
Confirmed all tests pass on my side and linux OS, thanks!